landing-page/content/common/table.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Iceberg Tables"
+url: concepts/table
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Iceberg Tables
+
+## Overview
+
+Tables are the central element in Iceberg. A table is a dataset with a common set of features (fields or columns) for each record (row). In addition to data files, tables contain a wealth of metadata information that enables many of Iceberg’s features such as schema evolution, fast query planning, snapshots for time travel, logical partitioning, and properties for optimizing table layout. Table properties are used to fine-tune each feature and allow you to control things such as the size when combining data input splits or the compression codec used when writing files.
+
+## Schema
+
+The schema of a table defines the elements contained in a single row. Each field in a schema can be optional or required and has an associated ID, name, and type. Schemas can also include aliases for certain fields. As the schema of an Iceberg table is changed, the new schema is given a unique ID. Past schemas are preserved in the table metadata and allow schemas to be restored when tables are rolled back.
```

Comment:
Rather than `elements`, I think this should be `fields or columns` to keep the language consistent.

I'm also not sure that I would keep this here. Schemas are important, but this seems to be going into detail about what Iceberg tracks, which isn't very valuable. I'd probably cover these things as sections, but as terms or things that you need to know about:

```
* Schema -- the fields that are tracked for every record, along with types and documentation
* Partition spec -- how to organize records in the table for fast access
* Write order -- how to cluster and order records in the table
* Snapshot -- a table version as of some point in time
```